src/data/restore.ts

```diff
@@ -189,8 +189,13 @@ export const restore = (
    */
   localAppState: Partial<AppState> | null | undefined,
 ): DataState => {
+  let scrollToCenter = true;
+  if (typeof data?.scrollToCenter !== "undefined") {
+    scrollToCenter = data.scrollToCenter;
+  }
   return {
     elements: restoreElements(data?.elements),
     appState: restoreAppState(data?.appState, localAppState || null),
+    scrollToCenter,
```

Comment:
`restore()` is used many times *after* init. Why not check `initialData.scrollToCenter` in `initializeScene`?